docs/platforms/react-native/user-feedback/index.mdx

```diff
@@ -1,9 +1,73 @@
 ---
 title: Set Up User Feedback
-description: "Learn more about collecting user feedback when an event occurs. Sentry pairs the feedback with the original event, giving you additional insight into issues."
+description: "Learn how to enable User Feedback in your app."
 sidebar_order: 6000
 ---
 
+The User Feedback feature allows you to collect user feedback from anywhere inside your application at any time, without needing an error event to occur first.
+
+Note that if you're using a self-hosted Sentry instance, you'll need to be on version 24.4.2 or higher in order to use the full functionality of the User Feedback feature. Lower versions may have limited functionality.
+
+## User Feedback Widget
+
+The user feedback widget allows users to submit feedback from anywhere inside your application.
+
+To collect user feedback from inside your application use the `showFeedbackForm` method.
+
+```javascript
+import * as Sentry from "@sentry/react-native";
+
+Sentry.showFeedbackForm();
+```
+
+<Alert>
+
+Note that the `showFeedbackForm` method depends on the React Native `Modal` implementation. It is supported fully in the legacy architecture. For the new architecture (Fabric renderer) it requires React Native `0.71` and up.
```

Comment:
Ref: https://github.com/getsentry/sentry-react-native/pull/4483#issuecomment-2613392055

Reply:
I think we can keep this only as a paragraph because the alert is quite intensive and pulls attention away from the snippet IMHO. And my expectation is only a small number of users should run into this.

Reply:
Good point 👍 Updated with 3144a50bebe19b69e0606e768759d38c6224a8e5